### PR TITLE
Add support for using global static variables

### PR DESCRIPTION
**Pull request by kant2002**
Case when static variable defined in the function does not handled yet.

### CODE REVIEW

**Review by ForNeVeR — CHANGES_REQUESTED** [review]

File: Cesium.CodeGen/Contexts/AssemblyContext.cs [hunk 1]
```diff
@@ -37,6 +37,7 @@ public class AssemblyContext
 
         var targetRuntime = compilationOptions.TargetRuntime;
         assembly.CustomAttributes.Add(targetRuntime.GetTargetFrameworkAttribute(module));
+        module.AssemblyReferences.Add(targetRuntime.GetSystemAssemblyReference());
```
Comment (ForNeVeR) [inline Cesium.CodeGen/Contexts/AssemblyContext.cs:40]: _(Minor)_ This line was removed in #385, but here it's back. I assume we should actually drop that?

The new tests work with it, though, so I assume it doesn't matter, and I merge :)

**Review by ForNeVeR — APPROVED** [review]
Thanks!

File: Cesium.CodeGen/Contexts/AssemblyContext.cs (hunk 1 above)
Comment (kant2002) [reply Cesium.CodeGen/Contexts/AssemblyContext.cs:40]: My general policy is to make all changes independent. and this change really was not important for global statics, so I only reduce changeset. It was a slip on my part to put removal in PR.

Comment (ForNeVeR) [reply Cesium.CodeGen/Contexts/AssemblyContext.cs:40]: Actually, it seems kinda a rebase/merge artifact rather than an actual change.

**Pull request merged**